frontend/src/routes/(authenticated)/org/[org_id]/+page.svelte

```diff
@@ -122,9 +136,19 @@
   <div class="py-6 px-2">
     {#if $queryParamValues.tab === 'projects'}
     <ProjectTable
+      canManage={canManage}
       columns={['name', 'code', 'users', 'type']}
       projects={org.projects}
-    />
+      on:removeProjectFromOrg={(event) => removeProjectFromOrg(event.detail.projectId, event.detail.projectName)}
+    >
+      <DeleteModal
```

Comment:
is there a reason the delete modal was placed inside the table? I'm not sure that's needed here. If it's not then move it outside of the table.

Reply:
It works outside of the table, but I followed how MembersList is done in `[project_code]\+page.svelte` as a model. It looks like DeleteModal is put inside of MembersList since the modal is unique to the list. Maybe for the sake of getting things organized?